src/flavors/__tests__/fixtures.ts

```diff
@@ -211,6 +211,11 @@ export function createFixtures(flavor: Flavor): Fixture {
             nullable: true,
         })
         nullableUUIDValue!: string | null;
+
+        @IsString({
+            name: 'string_value_with_different_wire_name',
```

Comment:
Opted to make this required in testing to ensure that we can always validate that it was converted correctly.